scripts/migrations/postgres/20211109185301_referral.up.sql

```diff
@@ -0,0 +1,88 @@
+CREATE EXTENSION IF NOT EXISTS "pgcrypto";
```

Comment:
I think we can just use hash of something. It's simpler than brute-forcing unique value. or use SERIAL + some salt

Reply:
The trigger generates a unique 8 letters code with a good selectivity. There is going to be only one loop in most cases.

Reply:
okay, shouldn't we have a ttl index for that column? when a referral's it doesn't make any sense

Reply:
What is the purpose of TTL? The referral code is not going to be expired, but the whole referral program might.